README.md

```diff
@@ -1 +1,40 @@
-# snips-react
\ No newline at end of file
+# snips-react
+
+This VS Code extension adds snippets for React in your editor for typescript.
+
+### TypeScript React Snippets (.tsx)
+
+| Snippet                      | Purpose                                     |
+|------------------------------|---------------------------------------------|
+| `react`                      | Snippet for default import (React & Radium) |
+| `store`                      | Snippet for store.dispatch                  |
+| `saga`                       | Snippet for Saga (watcher & worker)         |
+| `arrow`                      | Snippet for Arrow Function                  |
+| `it`                         | Snippet for it (testing)                    |
+| `unroll`                     | Snippet for unroll (testing)                |
+| `describe`                   | Snippet for describe (testing)              |
+
+### TypeScript Snippets (.ts)
+
+| Snippet                      | Purpose                                     |
+|------------------------------|---------------------------------------------|
+| `testsaga`                   | Snippet for testing Saga                    |
+
+Alternatively, press `Ctrl`+`Space` (Windows, Linux) or `Cmd`+`Space` (OSX) to activate snippets from within the editor.
+
+## Installation
+
+1. Install Visual Studio Code 1.19.0 or higher
+2. Launch Code
+3. From the command palette `Ctrl`-`Shift`-`X` (Windows, Linux) or `Cmd`-`Shift`-`X` (OSX)
+4. Select `Install Extension`
+5. Choose the extension
+6. Reload Visual Studio Code
+
+
+## Contribute
+
+This is an open source project and if you want to contribute, feel free to create issue in our [repository](https://github.com/causecode/snips-react) or create pull requests.
+
+
```

Comment:
Add a block for the `License` and just display `MIT` in it's body

Reply:
Done